api/RestfulService.php

```diff
@@ -67,6 +67,22 @@ public static function set_default_curl_options($optionArray) {
 	}
 
 	/**
+	 * Parses cURL options defined in YML configuration.
+	 *
+	 * @return array
+	 */
+	public static function get_default_curl_options() {
+		$rawOptions = Config::inst()->get('RestfulService', 'default_curl_options');
```

Comment:
Should use `get_called_class()` here instead of hardcoding in RestfulService, IMHO. Allows you to extend easier without needing to convert this in to an instance method and lookup using `$this->config()`.